scrc/dataset_creation/dataset_creator.py

```diff
@@ -156,15 +158,15 @@ def __init__(self, config: dict, debug: bool = True):
 
         self.debug = debug
         self.seed = 42
-        self.minFeatureColLength = 10  # tokens
+        self.minFeatureColLength = 100  # characters
         self.debug_chunksize = 100
         self.real_chunksize = 1_000_000
         self.counter = 0
-        self.start_years = {Split.TRAIN.value: 2002, Split.VALIDATION.value: 2016, Split.TEST.value: 2018,
-                            Split.SECRET_TEST.value: 2020}
+        self.start_years = {Split.TRAIN.value: 2002, Split.VALIDATION.value: 2016, Split.TEST.value: 2018}
```

Comment:
Why is the start year 2002 default for every dataset?

Reply:
For the bge criticality dataset we set the start year 2002 because before we do not have a complete dataset, which could lead to wrong results when setting the labels.
I'm not sure if we choose 2000 for all other datasets?

Reply:
_

> bis und mit Ende 2022 (Train bis und 2015, Validation 2016-2017, Test 2018-2022)

_ 
That's what we defined in the google doc.